src/components/views/homepage/introBlock/index.tsx

```diff
@@ -186,6 +185,20 @@ const IntroContainer = styled.div`
 	padding: 0 8px;
 `;
 
+const ContainerStyled = styled(Container)`
+	position: relative;
+	${mediaQueries.tablet} {
+		padding: 0;
+	}
+`;
```

Comment:
Thanks @RamRamez 
It's our standard grid, we don't need to remove its padding.
```suggestion
const ContainerStyled = styled(Container)`
	position: relative;
`;
```

Reply:
I fixed tablet screen size

![image](https://user-images.githubusercontent.com/73441857/223173093-b4f28d60-b648-4c70-a020-79a9dd708dac.png)

it's according to figma
https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/GIVeconomy?node-id=17955%3A234602&t=PCNLkDk91cgonIv8-0